client/src/main/java/tds/assessment/Segment.java

```diff
@@ -143,7 +168,7 @@ public void setFieldTestMinItems(int fieldTestMinItems) {
     /**
      * @return the maximum number of field test items in the {@link tds.assessment.Segment}
      */
-    public int getFieldTestMaxItems() {
+    public Integer getFieldTestMaxItems() {
```

Comment:
In fact, it looks like the loader defaults these values to "0", so although its not enforced by the DB, I don't think it's totally fine to treats these as primitives. 

ftstartpos/ftendpos on the other hand are typically null for all assessments first loaded (even the ones that have min/max field test item counts defined).